### DISCUSSION TITLE
YoLo Achievement

### DISCUSSION THREAD

**Discussion by mdAshik922** [community/community#38050] [Other Feature Feedback, Questions, & Ideas]
আজ আমি খুব সহজে **ইয়োলো অ্যাচিভমেন্ট** অর্জন করতে পেরেছি, আশাকরি দক্ষতার সাথে বাকি অ্যাচিভমেন্ট গুলো অর্জন করতে পারব।
Today I managed to get the **YOLO achievement** very easily. Hopefully, I can achieve the rest of the achievements efficiently.

**Comment by pascalfarrel** [comment]
Good for you bud. Keep grinding man

**Reply by mdAshik922** [reply]
thank you, bro

**Comment by david-engelmann** [comment]
PR without a review, 1 more PR and you'll get another!

**Comment by ossanna16** [comment]
Well done @mdAshik922! Welcome to our community!
